tests/main/refresh-app-awareness/task.yaml

```diff
@@ -19,11 +18,11 @@ restore: |
     snap remove --purge test-snapd-refresh
     rm -f test-snapd-refresh-{1,2}_all.snap
     rm -f test-snapd-refresh.*/meta/snap.yaml
-    rmdir /sys/fs/cgroup/pids/snap.test-snapd-refresh.sh || true
-    rmdir /sys/fs/cgroup/pids/snap.test-snapd-refresh.version || true
-    systemctl --user stop dbus.service || true
     snap unset core experimental.refresh-app-awareness
     rm -f install.log
+    if [ "$USER" = root ]; then
```

Comment:
this probably merits a comment about why/what we are doing here only for USER==root

Reply:
Agreed. I've reused the comment I wrote for the lxd task above.